addons/event/views/event_views.xml

```diff
@@ -9,116 +9,63 @@
-                                            <field name="tag_ids" widget="many2many_tags" options="{'color_field': 'color', 'no_quick_create': True}"/>
-                                        </div>
-                                    </div>
-                                </div>
-                            </div>
-                        </div>
-                        <h2>Attendees</h2>
-                        <div class="row mt16 o_settings_container" name="event_type_attendees">
-                            <div class="col-12 o_setting_box" name="event_type_attendees_tickets">
-                                <div class="o_setting_left_pane">
-                                    <field name="use_ticket"/>
-                                </div>
-                                <div class="o_setting_right_pane">
-                                    <label for="use_ticket"/>
-                                    <div class="row mt16" attrs="{'invisible': [('use_ticket', '=', False)]}">
-                                        <div class="col">
-                                            <field name="event_type_ticket_ids"
-                                                class="w-100"
-                                                context="{
-                                                    'tree_view_ref': 'event.event_type_ticket_view_tree_from_type',
-                                                    'form_view_ref': 'event.event_type_ticket_view_form_from_type'
-                                                }"/>
-                                        </div>
-                                    </div>
-                                </div>
-                            </div>
-                            <div class="col-12 col-lg-6 o_setting_box" name="event_type_attendees_auto_confirm">
-                                <div class="o_setting_left_pane">
-                                    <field name="auto_confirm"/>
-                                </div>
-                                <div class="o_setting_right_pane">
-                                    <label for="auto_confirm"/>
-                                    <div class="row">
-                                        <div class="col-lg-8 mt16 text-muted">
-                                            Events and registrations will automatically be confirmed
-                                            upon creation, easing the flow for simple events.
-                                        </div>
-                                    </div>
-                                </div>
-                            </div>
+                        <div class="oe_title" name="event_type_title">
+                            <label for="name" string="Event Template"/>
+                            <h1><field name="name" placeholder="e.g. Online Conferences" class="mb-2"/></h1>
                         </div>
+
+                       <group style="margin-top: 16px">
```

Comment:
Please avoid inline css. Why is it required ?

Reply:
Actually this is not required, it has been removed.